### ISSUE TITLE
"www." doesn't resolve

### ISSUE THREAD

**Issue by benceruleanlu** [benceruleanlu/Team-Website#32]
Issue should be resolved during our transfer to Cloudflare.

Currently waiting on DNS changes to propagate.

**Comment by benceruleanlu** [comment]
Solved, however, this may be triggering "too many redirect" errors.

That's another issue though, so this will be closed.
